solana-programs/reward-manager/program/src/processor.rs

```diff
@@ -290,29 +317,31 @@ impl Processor {
         assert_owned_by(reward_manager_info, program_id)?;
         assert_owned_by(sender_info, program_id)?;
 
+        // Retrieve the sender account, assert that 
+        // it's reward_manager field is this reward_manager.
```

Comment:
NIT - should this be its? also might reword as 'reward_manager field associated with this transaction' or something